assets/src/shared/lib/notify-error.ts

```diff
@@ -1,4 +1,8 @@
 import { toastManager } from '~/shared/lib/toastManager';
 
-export const notifyError = (err: any) =>
-  toastManager.error(err.message ?? 'Something went wrong: ' + '\n\n' + err.response.data.detail ?? 'No details');
+export const notifyError = (err: any) => {
+  const heading = err.response.data.title ?? err.message ?? 'Something went wrong: '
+  const details = err.response.data.detail ?? 'No details'
```

Comment:
maybe empty string?

Reply:
Hmm

Reply:
Let's try this

Reply:
fix b1654cfaf7acb94c1128dbb025d2b7e1c7ca3f29